### PR TITLE
GUACAMOLE-623: Document support for Kubernetes.

### PR DESCRIPTION
**Pull request by mike-jumper**

### CODE REVIEW

File: src/chapters/docker.xml [hunk 1]
```diff
@@ -47,10 +47,10 @@
     <section xml:id="guacd-docker-image">
         <title>Running the <package>guacd</package> Docker image</title>
         <para>The <package>guacd</package> Docker image is built from the released
-                <package>guacamole-server</package> source with support for VNC, RDP, SSH, and
-            telnet. Common pitfalls like installing the required dependencies, installing fonts for
-            SSH or telnet, and ensuring the FreeRDP plugins are installed to the correct location
-            are all taken care of. It will simply just work.</para>
+                <package>guacamole-server</package> source with support for VNC, RDP, SSH, telnet,
+            and Kubernetes. Common pitfalls like installing the required dependencies, installing
+            fonts for SSH or telnet, and ensuring the FreeRDP plugins are installed to the correct
```
Comment (necouchman) [inline src/chapters/docker.xml]: "fonts for SSH, telnet, or Kubernetes"?

File: src/chapters/installing.xml [hunk 2]
```diff
@@ -452,6 +461,31 @@
                                 </informaltable>
                             </entry>
                         </row>
+                        <row>
+                            <entry><link xl:href="https://github.com/seanmiddleditch/libtelnet"
+                                    >libwebsockets</link></entry>
```
Comment (necouchman) [inline src/chapters/installing.xml]: Is this for `libtelnet` or `libwebsockets`?  Link is to `libtelnet`, but link text is for `libwebsockets`??

Comment (mike-jumper) [reply src/chapters/installing.xml]: Whoops. Copypasta. It's supposed to be for libwebsockets.

File: src/chapters/docker.xml (hunk 1 above)
Comment (mike-jumper) [reply src/chapters/docker.xml]: Sure.

**Comment by mike-jumper** [comment]
OK - the above should be addressed.

File: src/chapters/installing.xml [hunk 3]
```diff
@@ -452,6 +461,31 @@
                                 </informaltable>
                             </entry>
                         </row>
+                        <row>
+                            <entry><link xl:href="https://libwebsockets.org/"
+                                >libwebsockets</link></entry>
```
Comment (necouchman) [inline src/chapters/installing.xml]: Sorry, this makes me ever so slightly twitchy.  Any reason not to have the closing angle bracket on the line with the rest of that opening tag?

So sorry, just a nitpick...

Comment (mike-jumper) [reply src/chapters/installing.xml]: I can move it up. It's just the way that the oXygen editor autoformatted the XML.

**Comment by mike-jumper** [comment]
Nit: removed.

I'm more and more leaning toward migrating to readthedocs, or some similar format which is not XML.

**Pull request merged**